.github/scripts/get-changelog-from-change-txt.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const marked = require("marked");
+const fs = require('fs');
+const { VERSION } = process.env;
+
+if (!VERSION) {
+	console.error('missing VERSION env var');
+	process.exit(1);
+	return;
+}
+
+for (let fileName of ['changelog', 'readme'] ) {
```

Comment:
wouldn't it be clearer if we just specified the full file names (`'changelog.txt', 'readme.txt'`) here?

Reply:
Yes it would! Done